### PR TITLE
created results page for quiz app

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
Kindly review the results page and let me know what do you think about it !
image
image

### CODE REVIEW

**Comment by mmanjuprasad** [comment]
The result page looks good and can be merged.

File: ResultPage/ResultPage.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    height: 100vh;
+    padding-top: 3%;
+}
+
+.home-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    gap:20px;
```
Comment (yaswanthmyneni) [inline ResultPage/ResultPage.css:18]: use rems more than 16px.

**Review by yaswanthmyneni — COMMENTED** [review]
looks good but on the results page, we also need to show the correct answers for the questions. It is mentioned in lms.

File: ResultPage/ResultPage.css (hunk 1 above)
Comment (kgajriaAgarwal) [reply ResultPage/ResultPage.css:18]: ok!

**Comment by Rishabh-078** [comment]
Code & UI looks great. Can be merged

**Pull request merged**